src/coprocessor/dag/executor/index_scan.rs

```diff
@@ -422,16 +422,15 @@ mod test {
         let mut wrapper = IndexTestWrapper::new(unique);
         wrapper.scan.set_desc(true);
 
-        let r1 = get_idx_range(TABLE_ID, INDEX_ID, i64::MIN, 0, unique);
-        let r2 = get_idx_range(TABLE_ID, INDEX_ID, 0, (KEY_NUMBER / 2) as i64, unique);
-        let r3 = get_idx_range(
+        let r1 = get_idx_range(TABLE_ID, INDEX_ID, 0, (KEY_NUMBER / 2) as i64, unique);
+        let r2 = get_idx_range(
             TABLE_ID,
             INDEX_ID,
             (KEY_NUMBER / 2) as i64,
             i64::MAX,
             unique,
         );
-        wrapper.ranges = vec![r1, r2, r3];
+        wrapper.ranges = vec![r1, r2];
```

Comment:
Why change this?

Reply:
Since the previous `r1` has been wrongly encoded.